public/Aframe/samples/loading-scene-1/scene-1.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <title>Loading Room 1</title>
+        <meta name="description" content="Loading Room 1" />
+        <script src="https://aframe.io/releases/1.1.0/aframe.min.js"></script>
```

Comment:
The physics engine looks like is broken on aframe 1.2.0, when I tried it locally, this seems to break. Have you tried using 1.2.0?
https://github.com/n5ro/aframe-physics-system/issues/187

Reply:
You are right, it seems like geometry is no longer defined by three.js and I just pass through the floor. I am not exactly sure how to approach this - doesn't this affect everything else that wants to use physics as well?